subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -37,6 +37,15 @@ config BT_LLL_VENDOR_NORDIC
 	select BT_CTLR_XTAL_ADVANCED_SUPPORT
 	select BT_CTLR_SCHED_ADVANCED_SUPPORT
 	select BT_CTLR_TIFS_HW_SUPPORT
+	# Direction Finding
+	select BT_CTLR_DF_CTE_TX_SUPPORT if HAS_HW_NRF_RADIO_BLE_DF && \
```

Comment:
nothing selects HAS_HW_NRF_RADIO_BLE_DF, so these options do not become available